Donut 2 Medbay Modernization

t-toasted opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Gigantic modernization of Donut 2's medbay. Majorly: adjusts the entry way, downsizes and moves genetics, removes thindows, adds in appropriate clothing lockers. Still retains the design beats from the original design, such as the layout, but updates it for a modern player. 


## Why's this needed? <!-- Describe why you think this should be added to the game. -->
Ensuring maps are modernization and playable is a good thing; Donut 2 may not be playable by default, but the few times it is played mean that having the various departments be modernization would make it much more enjoyable.


## Changelog
<!-- If necessary, put your changelog entry below. Otherwise, please delete it.
Use however you want to be credited in the changelog in place of CodeDude.
Use (*) for major changes and (+) for minor changes. For example: -->

```changelog
(u)Roddy
(*)Updates Donut2's medbay/genetics area. 
```

![image](https://user-images.githubusercontent.com/105123457/177434271-a9af5a47-9378-4c56-b388-45a2eb410c4d.png)

ZeWaka commented:
changelog not necessary since this is not a map in rotation

Sord213 commented:
Okay got a few things.
![Screenshot_302](https://user-images.githubusercontent.com/71185626/177439285-6120338b-7aff-4a5b-8372-f9743ab7b315.png)
Why the wires going through the wall? There's no need for a power connection there.
The black wires should not connect to the same grid as the red wires. Black wires are direct engine output for PTL. Congrats, you hotwired medbay by default.
The portrait and fire alarm should be offset to sit on the wall, not place directly on top. As it stands they're both viewable from inside the MDir's office and in maint.
![Screenshot_303](https://user-images.githubusercontent.com/71185626/177439286-c7e67749-722c-4aee-b20b-73d4c294c6e4.png)
Same for this portrait and fire alarm as above.
There are blinds but no switches to control them. Should probably also split the treatment rooms into their own areas if you plan on using area blind switches.
No. No captain's bedsheet in the monkey dome.
![Screenshot_304](https://user-images.githubusercontent.com/71185626/177439289-881bdd40-085b-4670-ba6c-da521b6ff24b.png)
Generally I'd avoid putting APCs on windows. Looks better on walls.
The button to open the medbay doors can't be accessed while sitting on either of the reception chairs. Should probably move it to the tile in between the two chairs so they both can access it and offset it onto the window.
![Screenshot_305](https://user-images.githubusercontent.com/71185626/177439290-04f212e0-0a3a-413c-ae6f-42cdf48f245e.png)
The pipes and manifolds here are layered incorrectly and go under the floors, where the two by the freezer/connector do not. 
No again. No psychedelic bedsheets chilling in medbay's spare room.
![Screenshot_306](https://user-images.githubusercontent.com/71185626/177439291-efa3e2e6-f46f-4916-8da3-27cabd4056a6.png)
This whole maintenance area is bad. Open and empty and bad. Condense it a little, connect the two bits. Move some of the kitchen pipes if you have to.

Few other notes:
While funny, let's not put a knife in the monkey dome to murder the geneticists with.
The button in genetics does nothing.
There are errant wires and pipes under the doors in the morgue.
In maint behind robotics, the roboticists don't need two free crates of borg parts. The one (robotics_supplies) has their spare materials and should generally be inside robotics so it doesn't get stolen.
Why does the Medical Director have a Heisenbee plush and Research Director have a Morty plush? 
Please check all fire alarms and wall decorations/objects to make sure they're offset properly and not directly on top of walls unless they're meant to be visible from multiple angles and rooms.
Add some tile colors. Almost all of medbay is plain white and boring. Edge tile colors or other designs.

Sord213 reviewed (changes requested):
Okay got a few things.
![Screenshot_302](https://user-images.githubusercontent.com/71185626/177439285-6120338b-7aff-4a5b-8372-f9743ab7b315.png)
Why the wires going through the wall? There's no need for a power connection there.
The black wires should not connect to the same grid as the red wires. Black wires are direct engine output for PTL. Congrats, you hotwired medbay by default.
The portrait and fire alarm should be offset to sit on the wall, not place directly on top. As it stands they're both viewable from inside the MDir's office and in maint.
![Screenshot_303](https://user-images.githubusercontent.com/71185626/177439286-c7e67749-722c-4aee-b20b-73d4c294c6e4.png)
Same for this portrait and fire alarm as above.
There are blinds but no switches to control them. Should probably also split the treatment rooms into their own areas if you plan on using area blind switches.
No. No captain's bedsheet in the monkey dome.
![Screenshot_304](https://user-images.githubusercontent.com/71185626/177439289-881bdd40-085b-4670-ba6c-da521b6ff24b.png)
Generally I'd avoid putting APCs on windows. Looks better on walls.
The button to open the medbay doors can't be accessed while sitting on either of the reception chairs. Should probably move it to the tile in between the two chairs so they both can access it and offset it onto the window.
![Screenshot_305](https://user-images.githubusercontent.com/71185626/177439290-04f212e0-0a3a-413c-ae6f-42cdf48f245e.png)
The pipes and manifolds here are layered incorrectly and go under the floors, where the two by the freezer/connector do not. 
No again. No psychedelic bedsheets chilling in medbay's spare room.
![Screenshot_306](https://user-images.githubusercontent.com/71185626/177439291-efa3e2e6-f46f-4916-8da3-27cabd4056a6.png)
This whole maintenance area is bad. Open and empty and bad. Condense it a little, connect the two bits. Move some of the kitchen pipes if you have to.

Few other notes:
While funny, let's not put a knife in the monkey dome to murder the geneticists with.
The button in genetics does nothing.
There are errant wires and pipes under the doors in the morgue.
In maint behind robotics, the roboticists don't need two free crates of borg parts. The one (robotics_supplies) has their spare materials and should generally be inside robotics so it doesn't get stolen.
Why does the Medical Director have a Heisenbee plush and Research Director have a Morty plush? 
Please check all fire alarms and wall decorations/objects to make sure they're offset properly and not directly on top of walls unless they're meant to be visible from multiple angles and rooms.
Add some tile colors. Almost all of medbay is plain white and boring. Edge tile colors or other designs.

t-toasted commented:
> Okay got a few things. ![Screenshot_302](https://user-images.githubusercontent.com/71185626/177439285-6120338b-7aff-4a5b-8372-f9743ab7b315.png) Why the wires going through the wall? There's no need for a power connection there. The black wires should not connect to the same grid as the red wires. Black wires are direct engine output for PTL. Congrats, you hotwired medbay by default. The portrait and fire alarm should be offset to sit on the wall, not place directly on top. As it stands they're both viewable from inside the MDir's office and in maint. ![Screenshot_303](https://user-images.githubusercontent.com/71185626/177439286-c7e67749-722c-4aee-b20b-73d4c294c6e4.png) Same for this portrait and fire alarm as above. There are blinds but no switches to control them. Should probably also split the treatment rooms into their own areas if you plan on using area blind switches. No. No captain's bedsheet in the monkey dome. ![Screenshot_304](https://user-images.githubusercontent.com/71185626/177439289-881bdd40-085b-4670-ba6c-da521b6ff24b.png) Generally I'd avoid putting APCs on windows. Looks better on walls. The button to open the medbay doors can't be accessed while sitting on either of the reception chairs. Should probably move it to the tile in between the two chairs so they both can access it and offset it onto the window. ![Screenshot_305](https://user-images.githubusercontent.com/71185626/177439290-04f212e0-0a3a-413c-ae6f-42cdf48f245e.png) The pipes and manifolds here are layered incorrectly and go under the floors, where the two by the freezer/connector do not. No again. No psychedelic bedsheets chilling in medbay's spare room. ![Screenshot_306](https://user-images.githubusercontent.com/71185626/177439291-efa3e2e6-f46f-4916-8da3-27cabd4056a6.png) This whole maintenance area is bad. Open and empty and bad. Condense it a little, connect the two bits. Move some of the kitchen pipes if you have to.
> 
> Few other notes: While funny, let's not put a knife in the monkey dome to murder the geneticists with. The button in genetics does nothing. There are errant wires and pipes under the doors in the morgue. In maint behind robotics, the roboticists don't need two free crates of borg parts. The one (robotics_supplies) has their spare materials and should generally be inside robotics so it doesn't get stolen. Why does the Medical Director have a Heisenbee plush and Research Director have a Morty plush? Please check all fire alarms and wall decorations/objects to make sure they're offset properly and not directly on top of walls unless they're meant to be visible from multiple angles and rooms. Add some tile colors. Almost all of medbay is plain white and boring. Edge tile colors or other designs.

Hehe medbay time. 

Give me… a week and I’ll have it updated with all of this modified.

t-toasted commented:
@Sord213 , everything but the pipes in cryo have been fixed, I believe. Need some help with that, but, itemized list:

-fixed wire in MD office
-offsets all portraits/wall decals/fire alarms
-removed blinds
-removed captain's sheet from Monkeydome
-apc removed from window and put onto metal wall
-button offset properly
-did NOT know that 'psych' stood for psychedelic, OOOOPSIE
-Fixes maint area. I plan on editing the kitchen later on sometime but it's nice for now

SMALLER CHANGES
-knife removed
-button removed
-removed errant pipe in morg. Not sure where errant wire is though
-removed one set of borg crates, placed other one into robotics
-MD and RD have respective plushes because 1) thought it would be cute 2) on most maps, chemistry and medical work closely together. RD has access to the medical channel too. They should be friends (:
-adds more color and variants to the tile colors

Sord213 reviewed (changes requested):
![Screenshot_316](https://user-images.githubusercontent.com/71185626/178333508-e2015e9a-9446-49f5-a3a6-9278db00ca71.png)
Power cables are still connected.
